### ISSUE TITLE
Plugins don't return results when TSLint is installed globally

### ISSUE THREAD

**Issue by ksdmitrieva** [palantir/tslint#4015]
### Bug Report

- __TSLint version__: 5.10.0
- __TypeScript version__: 1.0.3.0
- __Running TSLint via__: CLI on Windows

When TSLint is installed globally, but a TSLint plugin is installed locally, the plugin doesn't return any results. And TSLint also doesn't show any error messages. Tested this on multiple projects with two plugins:
* tslint-microsoft-contrib https://github.com/Microsoft/tslint-microsoft-contrib
* tslint-eslint-rules https://github.com/buzinas/tslint-eslint-rules/issues

I intentionally have issues in the code that must be reported by these two plugins

#### TypeScript code being linted

```ts
    const ran = Math.random().toString();
    if (true) {
      document.cookie = 'Session=' + ran;
      console.log("Always true "+ ran);
    }
```

with `tslint.json` configuration:

```json
{
  "rulesDirectory": [
	"node_modules/tslint-microsoft-contrib",
	"node_modules/tslint-eslint-rules"
  ],

  "rules": {
       "insecure-random": true,
  	"no-cookies": true,

       "no-constant-condition": true,
  }
}
```
Running with command: tslint --project tsconfig.json --config tslint.json

#### Actual behavior
No issues are reported if TSLint is installed globally and plugins are installed locally.

#### Expected behavior
Three issues mut be reported:
* insecure random (Math.random())
* constant condition (if (true))
* cookies used (document.cookie)

If TSLint is installed locally in the project, the issues are reported as expected. 

If TSLint is installed globally and the plugins are installed globally, TSLint returns the following error: 
Failed to load C:\[path]\tslint.json: Could not find custom rule directory: node_modules/tslint-microsoft-contrib

If TSLint is installed globally but the plugins are installed locally, no results for the plugins are reported (the standard TSLint rules work as expected) and no error messages are displayed.

I suggest to change the ReadMe saying that to run TSLint with plugins, it has to be installed locally and the plugins must be installed locally.

**Comment by johnwiseheart** [comment]
This is a pretty standard javascript thing - the `node_modules` folder for globally installed packages and locally installed packages is in a different place and they do not talk to eachother. You should in theory be able to install everything globally, or everything locally, but it will not work if you do some globally and some locally.
